nfe/wizard/nfe_xml_periodic_export.py

```diff
@@ -0,0 +1,200 @@
+# -*- encoding: utf-8 -*-
```

Comment:
👎replace former header format

Replace # -- encoding: utf-8 -- for # -- coding: utf-8 --
https://www.python.org/dev/peps/pep-0263/#examples